src/utils/router.ts

```diff
@@ -9,7 +9,11 @@ export const safeAddressFromUrl = (): string => {
   })
 
   if (match) {
-    return checksumAddress(match.params.safeAddress).toString()
+    try {
+      return checksumAddress(match.params.safeAddress).toString()
+    } catch (err) {
+      throw new Error(err.message)
```

Comment:
It's the same thing as not catching it at all 🙈

Reply:
Brain fart. I don't think we need to catch this anyway:

![image](https://user-images.githubusercontent.com/20442784/135081607-5b48b91e-d622-41cf-8ab8-58f157f5134d.png)

The Error Boundary does for us:

![image](https://user-images.githubusercontent.com/20442784/135081558-3f41ef3f-98b9-40c4-af7a-536c438c6f90.png)